Peer.Domain/IPullRequestSource.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Peer.Domain.Models;
+
+namespace Peer.Domain
+{
+    public interface IPullRequestSource
+    {
+        Task<IEnumerable<PeerPullRequest>> FetchPullRequests();
```

Comment:
Async methods in C# have Async appended

Reply:
We can't have async inside interfaces. Async needs a method body with await statements.

Reply:
All async methods* have async appended to their MethodName* so this would be FetchPullRequestsAsync